### PR TITLE
Add connectionPool and test

### PR DESCRIPTION
**Pull request by laura-ding**

### CODE REVIEW

File: client/src/main/java/com/vesoft/nebula/client/graph/GraphClient.java [hunk 1]
```diff
@@ -1,46 +0,0 @@
-/* Copyright (c) 2019 vesoft inc. All rights reserved.
- *
- * This source code is licensed under Apache 2.0 License,
- * attached with Common Clause Condition 1.0, found in the LICENSES directory.
- */
-
```
Comment (darionyaphet) [inline client/src/main/java/com/vesoft/nebula/client/graph/GraphClient.java:6]: why remove Graph Client ? They are not conflict ？

**Comment by darionyaphet** [comment]
Actually IMO you can get `GraphClient` from the connection pool .

File: client/src/main/java/com/vesoft/nebula/client/graph/net/Connection.java [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+import com.facebook.thrift.transport.TTransportException;
+import com.vesoft.nebula.client.graph.data.HostAddress;
+import com.vesoft.nebula.client.graph.exception.AuthFailedException;
+import com.vesoft.nebula.client.graph.exception.IOErrorException;
+import com.vesoft.nebula.graph.AuthResponse;
+import com.vesoft.nebula.graph.ErrorCode;
+import com.vesoft.nebula.graph.ExecutionResponse;
+import com.vesoft.nebula.graph.GraphService;
+
+public class Connection {
```
Comment (darionyaphet) [inline client/src/main/java/com/vesoft/nebula/client/graph/net/Connection.java:24]: Is it similar with GraphClient ?

**Comment by darionyaphet** [comment]
https://www.commonpool.org/ maybe a good choice

File: client/src/main/java/com/vesoft/nebula/client/graph/net/Connection.java (hunk 2 above)
Comment (laura-ding) [reply client/src/main/java/com/vesoft/nebula/client/graph/net/Connection.java:24]: No, it just the connection, the GraphClient can use more than one connection if the connection has broken.

**Comment by darionyaphet** [comment]
refer [Add connection pool and LoadBalancer for java client](https://github.com/vesoft-inc/nebula-clients/pull/11)

**Pull request closed without merging**